api/vars.go

```diff
@@ -37,6 +37,7 @@ const (
 	DefaultReleaseHistoryAgeLimit           = 0
 	DefaultReleaseHistoryCountLimit         = 10
 	DefaultTimeoutSeconds                   = 30
+	DefaultEventListenerSize                = 1
```

Comment:
The default should probably be higher. CPU count * 2 is usually a good default.

```go
runtime.NumCPU() * 2
```

Reply:
Updated in https://github.com/xigxog/kubefox/pull/29/commits/cc9526c3a0e1fcfa278d5001efaa9dee2857fa72